src/Umbraco.ModelsBuilder/Umbraco.ModelsBuilder.csproj

```diff
@@ -110,14 +92,23 @@
     </PackageReference>
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\Umbraco.Abstractions\Umbraco.Abstractions.csproj">
```

Comment:
Hrm, this isn't a project ;) I'll fixup

Reply:
whoops.. (I tested on the unicore project, must have forgot to revert that change)